packages/toolpad-app/src/appDom/migrations/index.ts

```diff
@@ -1,9 +1,10 @@
 import invariant from 'invariant';
 import v1 from './v1';
 import v2 from './v2';
+import v3 from './v3';
 import * as appDom from '..';
 
-const versions = [v1, v2];
+const versions = [v1, v2, v3];
```

Comment:
There already is a v3, I had just forgotten to include it 🤦 
https://github.com/mui/mui-toolpad/pull/1359